### PR TITLE
[MTM-56086]: Release Notes for device access token

### PR DESCRIPTION
**Pull request by cumulocitypans**
https://cumulocity.atlassian.net/browse/MTM-56086
User documentation : https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication

### CODE REVIEW

**Comment by BeateRixen** [comment]
@cumulocitypans The PR title says that this is an operational release notes. If so it must go to the c8y-docs-ops repo and not to the public release notes. Please let me know if this is the case, and I will add it there.

**Comment by cumulocitypans** [comment]
> @cumulocitypans The PR title says that this is an operational release notes. If so it must go to the c8y-docs-ops repo and not to the public release notes. Please let me know if this is the case, and I will add it there.

@BeateRixen It is only for release notes, sorry for confusion, I will edit PR name.
I will discuss with team if it is needed for operational release notes then I will create seperate PR on `c8y-docs-ops` repo.

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2024-03-30T10:33:39.707Z
+title: Device access token
```
Comment (kalavathig1) [inline content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md:3]: can this be elobrated? Maybe something like `Device certificate authentication to retrieve access token`

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+change_type:
+  - value: change-QHu1GdukP
+    label: Feature
+product_area: Device management & connectivity
+component:
+  - value: component-1KLUzmqfe
```
Comment (kalavathig1) [inline content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md:9]: component value is `q3kclF6pO` and not `component-1KLUzmqfe`

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    label: X509 certificate authentication
```
Comment (kalavathig1) [inline content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md:10]: label is `Authentication` and not `X509 certificate authentication` 
Note: for all these product_area, component, value and label you can refer doc https://sagportal.sharepoint.com/:w:/r/sites/IoTAnalyticsRnDOps/_layouts/15/Doc.aspx?sourcedoc=%7B70D4BC16-3290-41B9-A515-ECF56FCBB958%7D&file=Release%20Notes%20creation%20process%202.0.docx&action=default&mobileredirect=true&ovuser=d9662eb9-ad98-4e74-a8a2-04ed5d544db6%2Ckgol%40softwareag.com&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yNDAyMjkyMDYwMiJ9 for any clarification please contact me

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2024-03-30T10:33:39.707Z
+title: Device access token
+change_type:
+  - value: change-QHu1GdukP
+    label: Feature
+product_area: Device management & connectivity
```
Comment (kalavathig1) [inline content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md:7]: its `Platform services` and not `Device management & connectivity`

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-1KLUzmqfe
+    label: X509 certificate authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-54838
+version: 10.20.327.0
```
Comment (kalavathig1) [inline content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md:15]: fixed versions here is 
image
and not 10.20.327.0. I think its typo please correct

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+---
+Devices get JWT session token with X.509 certificates via REST are now supported. For details, refer to the [Device authentication](https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication) user documentation.
```
Comment (kalavathig1) [inline content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md:17]: Could be `Added support to retrieve JWT session token with X.509 certificates via REST API for devices. For details, refer to the [Device authentication](https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication) user documentation.`

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2024-03-30T10:33:39.707Z
```
[the remark this answers is not in the document]
Comment (cumulocitypans) [reply content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md]: Date removed

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md (hunk 2 above)
Comment (cumulocitypans) [reply content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md]: changed to q3kclF6pO

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md (hunk 3 above)
Comment (cumulocitypans) [reply content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md]: changed

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md (hunk 5 above)
Comment (cumulocitypans) [reply content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md]: Corrected

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md (hunk 6 above)
Comment (cumulocitypans) [reply content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md]: Done

File: content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md (hunk 1 above)
Comment (cumulocitypans) [reply content/change-logs/device-management/cumulocity-10-20-327-0-X-509-device-access-token.md]: Changed

File: content/change-logs/platform-services/cumulocity-10-20-237-0-X-509-device-access-token.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-54838
+version: 10.20.237.0
+---
+Added support to retrieve JWT session token with X.509 certificates via REST API for devices. For details, refer to the [Device authentication](https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication) user documentation.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-237-0-X-509-device-access-token.md:17]:
```suggestion
Support has been added for retrieving JWT session tokens with X.509 certificates via REST API for devices. For details, refer to the [Device authentication](https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication) user documentation.
```

File: content/change-logs/platform-services/cumulocity-10-20-237-0-X-509-device-access-token.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device certificate authentication to retrieve JWT session token
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-237-0-X-509-device-access-token.md:3]:
```suggestion
title: Added support for retrieving JWT session token with X.509 certificates
```

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-237-0-X-509-device-access-token.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Added support for retrieving JWT session token with X.509 certificates
```
Comment (kalavathig1) [inline content/change-logs/platform-services/cumulocity-10-20-237-0-X-509-device-access-token.md:3]: Can this `Added support for retrieving JWT session token with X.509 certificates` be `Added support for retrieving JWT session token with X.509 certificates for devices` ?

Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-237-0-X-509-device-access-token.md:3]:
```suggestion
title: Added support for retrieving JWT session token with X.509 certificates for devices
```

Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10-20-237-0-X-509-device-access-token.md]: > Can this `Added support for retrieving JWT session token with X.509 certificates` be `Added support for retrieving JWT session token with X.509 certificates for devices` ?

Sure, changed it.

**Review by kalavathig1 — APPROVED** [review]

**Pull request merged**